tests/integration/test_integration.py

```diff
@@ -694,46 +694,6 @@ def test_regression_logging_in_context(tmpdir, logs_injection, debug_mode, patch
     assert p.returncode == 0
 
 
-@pytest.mark.parametrize(
-    "call_basic_config,debug_mode",
-    itertools.permutations((True, False, None), 2),
-)
-def test_call_basic_config(ddtrace_run_python_code_in_subprocess, call_basic_config, debug_mode):
```

Comment:
With the approach taken in this PR to unload `logging` (we don't expect it to be loaded at startup, so it would end up being cleaned up), there is no need for special handling, and this test would in fact fail. We should probably bring it back if we decide we don't want to unload `logging`.

Reply:
I think we can minimize this change's risk by only unloading the modules necessary to fix gunicorn compatibility as described in https://github.com/DataDog/dd-trace-py/pull/4863/files#r1096004330 . If we do that this change will become unnecessary.